core/network/fabric-interop-cc/libs/test/setup_test.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// test contains the stubs required for counterfeiter to generate the mocks used for tests
+//
+// The mocks are created by running `go generate ./...` as described in the counterfeiter readme
+package test
```

Comment:
@VRamakrishna, Ideally the package name here would be `test_test`. But that's not looking good. Can we rename the package `test` to `teststubs` and hence this package name would be `teststubs_test`. And rename the directory `libs/test` to `libs/teststubs` ?

Reply:
Thanks. I'll rename the folder to `testutils`. We may end up using it for more than stub creations.